jmxexporter-prometheus-grafana/assets/prometheus/jmx-exporter/kafka_connect.yml

```diff
@@ -84,14 +84,14 @@ rules:
     labels:
       connector: "$1"
       task: "$2"
-  # confluent.replicator:type=confluent-replicator-task-metrics,*
-  - pattern: "confluent.replicator<type=confluent-replicator-task-metrics,(.+)=(.+),(.+)=(.+),(.+)=(.+),(.+)=(.+)><>([^:]+)"
-    name: confluent_replicator_confluent_replicator_task_metrics_$9
+  # confluent.replicator:type=confluent-replicator-task-metrics,confluent-replicator-task-topic-partition-*
+  - pattern: "confluent.replicator<type=confluent-replicator-task-metrics,(.+)=(.+),(.+)=(.+),(.+)=(.+),(.+)=(.+)><>confluent-replicator-task-topic-partition-([^:]+)"
+    name: confluent_replicator_task_metrics_$9
```

Comment:
The naming convention that was put in place with CP 6.0.0 was to follow the format <BeanName>_<typeName>_<attributeName>. 
I understand it makes the name a bit longer, but makes it follow the same pattern everywhere. Will it be okay to follow th same approach ?

Reply:
I think the name is too verbose and it looks repeating itself. From prometheus standpoint I am not sure if that's efficient.